package.json

```diff
@@ -13,10 +13,14 @@
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
     "@mui/material": "^5.15.18",
+    "formik": "^2.4.6",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-hook-form": "^7.51.4",
```

Comment:
why this library is installed? I don't see its usage

Reply:
I was using UseForm Hook and then converted to Formik. I will delete it

Reply:
yes uninstall it, you should also uninstall "@hookform/devtools": "^4.3.1",